src/style.css

```diff
@@ -0,0 +1,150 @@
+.page-block {
+  margin: 0;
+
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  max-width: 200px;
+
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  align-self: center;
+
+  max-width: 160px;
+  height: 134px;
+
+  margin-bottom: 40px;
+}
+
+.card__inscription {
+  font-style: normal;
```

Comment:
```suggestion
```
this property is default. you can delete it.

Reply:
Ok!